app/commands.py

```diff
@@ -1407,6 +1407,12 @@ async def rmpriv(ctx: Context) -> Optional[str]:
     if not (t := await app.state.sessions.players.from_cache_or_sql(name=ctx.args[0])):
         return "Could not find user."
 
+    if bits & Privileges.DONATOR:
+        await app.state.services.database.execute(
+            "UPDATE users SET donor_end = 0 WHERE id = :user_id",
+            {"user_id": t.id},
+        )
```

Comment:
it may actually be more technically correct to have this functionality in the Player.remove_privs call itself, but i'm not too worried as this is all being refactored as we speak anyways.

Reply:
Alr